CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.1.2
+- Fixed wrapper not found error.
```

Comment:
Explain what use case was fixed. (That's much more useful for someone who doesn't know the internals of our project.)

Reply:
How about this?
- Fixed wrapper not found error when running `pub run ffigen`.

> Please run dartfmt on..

There was an extra newline at the end, this has been fixed in the PR. Running `dartfmt -w` says file unchanged.

Reply:
> Fixed wrapper not found error when running pub run ffigen.

👍 

> There was an extra newline at the end, this has been fixed in the PR. Running dartfmt -w says file unchanged.

My bad, I missed that when looking at the diff.